include/z64light.h

```diff
@@ -43,12 +43,15 @@ typedef struct LightNode {
     /* 0x8 */ struct LightNode* next;
 } LightNode; // size = 0xC
 
+#define LIGHTCTX_FOGNEAR_MAX 996
+#define LIGHTCTX_ZFAR_MAX 12800
+
```

Comment:
no need for ctx in the name imo, Light_ or Lights_ for the system name is identifying enough

Reply:
The prefix could be `ENV_` too? the values are in `LightContext` but they're computed in `Environment_Update` 😅